src/e2e/java/teammates/e2e/cases/e2e/InstructorCourseEnrollPageE2ETest.java

```diff
@@ -0,0 +1,100 @@
+package teammates.e2e.cases.e2e;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.e2e.pageobjects.AppPage;
+import teammates.e2e.pageobjects.InstructorCourseEnrollPage;
+import teammates.e2e.pageobjects.InstructorHomePage;
+
+/**
+ * SUT: {@link Const.WebPageURIs#INSTRUCTOR_COURSE_ENROLL_PAGE}.
+ */
+public class InstructorCourseEnrollPageE2ETest extends BaseE2ETestCase {
+    @Override
+    protected void prepareTestData() {
+        testData = loadDataBundle("/InstructorCourseEnrollPageE2ETest.json");
+        removeAndRestoreDataBundle(testData);
+    }
+
+    @Test
+    public void testAll() {
+        AppUrl url = createUrl(Const.WebPageURIs.INSTRUCTOR_COURSE_ENROLL_PAGE)
+                .withUserId(testData.instructors.get("ICEnrollE2eT.teammates.test").googleId)
+                .withCourseId(testData.courses.get("ICEnrollE2eT.CS2104").getId());
+        loginAdminToPage(url, InstructorHomePage.class);
+        InstructorCourseEnrollPage enrollPage =
+                AppPage.getNewPageInstance(browser, url, InstructorCourseEnrollPage.class);
+
+        ______TS("Add rows to enroll spreadsheet");
+        int numRowsToAdd = 80;
```

Comment:
80 is a bit too much, actually. We're not trying to load-test here, so a smaller number like 50 or even 30 should work.